### PR TITLE
Branch/feature1

### PR DESCRIPTION
**Pull request by simonysm5018**

### CODE REVIEW

File: hello1.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 hello1
 make change by simonysm5018
+make change by simonysm5018 2
```
Comment (simonysm) [inline hello1.txt]: good

File: hello1.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 hello1
 make change by simonysm5018
```
Comment (simonysm) [inline hello1.txt]: gd

**Review by simonysm — COMMENTED** [review]
test

File: hello1.txt [hunk 3]
```diff
@@ -1,2 +1,3 @@
 hello1
-make change by simonysm5018
+make change by simonysm5018 (some changes here).
+make change by simonysm5018 2 (some changes here).
```
Comment (simonysm) [inline hello1.txt:3]: change again.

File: hello2.txt [hunk 4]
```diff
@@ -5,3 +5,4 @@ hello2 by simonysm5018 2nd change (changed by simonysm5018)
 hello2d
 5th changed
 make change by simonysm
```
Comment (simonysm) [inline hello2.txt:7]: pls. change

File: hello4.txt [hunk 5]
```diff
@@ -1,2 +1,3 @@
 hello4
 make change by simonysm
```
Comment (simonysm) [inline hello4.txt:2]: change again

File: hello5.txt [hunk 6]
```diff
@@ -1 +1,2 @@
 simonysm5018 's change
+simonysm5018 's change 2
```
Comment (simonysm) [inline hello5.txt:2]: change again

File: hello1.txt (hunk 3 above)
Comment (simonysm5018) [reply hello1.txt:3]: changed

File: hello2.txt (hunk 4 above)
Comment (simonysm5018) [reply hello2.txt:7]: changed

File: hello4.txt (hunk 5 above)
Comment (simonysm5018) [reply hello4.txt:2]: changed.

File: hello5.txt (hunk 6 above)
Comment (simonysm5018) [reply hello5.txt:2]: changed.

**Review by simonysm — APPROVED** [review]

**Pull request merged**